### PR TITLE
Android app - Better actionable error messages

### LINKED ISSUE #419: [Android App] Sending OTP failed with reason null
On Android app co retriever shows sending OTP failed with reason null and continuously try it ..
But failed

### PR DESCRIPTION
**Pull request by chris-10**
Fixed #419 - Updated app to display definite actionable error messages. Eliminated any possibility of a 'null' error message.

### CODE REVIEW

**Comment by bombardier-gif** [comment]
@BumpyKnuckls FYI

File: src/CoWinOTPRetriever/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -13,7 +13,14 @@
     <string name="otp_send_success">Successfully sent CoWIN OTP: %1$s</string>
     <string name="send_otp_fail_message">Sending OTP failed with reason: %1$s. Retry attempt #%2$d..</string>
     <string name="otp_received_toast">Sending OTP to %1$s from CoWIN: %2$s</string>
-    <string name="generic_server_down">KVDB server is down</string>
-    <string name="no_internet">No internet connection</string>
-    <string name="generic_error">An unknown error occurred</string>
+    <string name="error_01">Server is not connected to the internet</string>
```
Comment (BumpyKnuckls) [inline src/CoWinOTPRetriever/app/src/main/res/values/strings.xml:16]: Can you please make the string variable names more descriptive? error_01 -> error_server_not_connected
error_02 -> device_not_connected,  error_07 -> error_kvdb_unresponsive and so on...

**Review by BumpyKnuckls — CHANGES_REQUESTED** [review]
Left a comment. Will leave more comments if required

File: src/CoWinOTPRetriever/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (chris-10) [reply src/CoWinOTPRetriever/app/src/main/res/values/strings.xml]: Done

**Comment by BumpyKnuckls** [comment]
Thanks @chris-10 for doing this. Landing this one.

**Pull request merged**